pkg/autoscaler/config/config.go

```diff
@@ -131,16 +135,24 @@ func validate(lc *autoscalerconfig.Config) (*autoscalerconfig.Config, error) {
 		return nil, fmt.Errorf("target-burst-capacity must be either non-negative or -1 (for unlimited), was: %f", lc.TargetBurstCapacity)
 	}
 
-	if lc.ContainerConcurrencyTargetFraction <= 0 || lc.ContainerConcurrencyTargetFraction > 1 {
+	if lc.ContainerConcurrencyTargetFraction < 0 || lc.ContainerConcurrencyTargetFraction > 1 {
 		return nil, fmt.Errorf("container-concurrency-target-percentage = %f is outside of valid range of (0, 100]", lc.ContainerConcurrencyTargetFraction)
```

Comment:
```suggestion
		return nil, fmt.Errorf("container-concurrency-target-percentage = %f is outside of valid range of (0, 1]", lc.ContainerConcurrencyTargetFraction)
```

Reply:
Thanks, done. This should be `[0, 100]` because if `container-concurrency-target-percentage` is greater than 1 it will be divided by 100.